contracts/LifMarketMaker.sol

```diff
@@ -63,21 +68,25 @@ contract LifMarketMaker is Ownable {
     totalPeriods = _totalPeriods;
     foundationAddr = _foundationAddr;
     initialWei = msg.value;
-    initialBuyPrice = initialWei.div(lifToken.totalSupply());
+    initialBuyPrice = initialWei
+      .div(lifToken.totalSupply().div(PRICE_FACTOR))
+      .mul(PRICE_FACTOR);
+    initialSellPrice = initialBuyPrice.div(PRICE_FACTOR).mul(initialPriceSpread);
```

Comment:
initiatlSpread its already mulltiplied by PRICE_FACTOR, and yes, I will change the order.